### ISSUE TITLE
How to draw Figure 9-11 in the paper?

### ISSUE THREAD

**Issue by xqoby** [ReedOnePeck/MindDiffuser#13]

**Comment by mind-animator-design** [comment]
Once the linear decoding models were trained, the weights at each voxel represent the importance of that voxel in decoding a specific feature. For example, if the number of voxels in the training set is 𝑀 and the feature dimension is 𝑁, then the decoder weights form an 𝑀×𝑁 matrix. By averaging this matrix along the N-dimensional axis, we obtain a vector of shape (𝑀,  ). Using the software pycortex (https://github.com/gallantlab/pycortex), this vector can be projected onto the cortex to generate Figures 9, 10, and 12.

I hope this explanation can help you.
